charts/incubator/alfresco-infrastructure/templates/pvc.yaml

```diff
@@ -1,34 +1,34 @@
-{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
-{{- if eq .Values.persistence.volumeEnv "aws" -}}
+{{- if and .Values.persistence.efs.enabled (not .Values.persistence.existingClaim) -}}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```

Comment:
Do we still need to specify a storage class?

Reply:
We don't really need to but it does prevent a dashboard error when you want to see volumes created using this class.

Reply:
Do we still need to create volumes under a specific dummy storage class?

Reply:
Sadly if we dont speficy a class for the PVC the default one is picked up so then it tries to bind to gp2 instead of the volume created.

Reply:
OK